Formula/n/newsraft.rb

```diff
@@ -0,0 +1,28 @@
+class Newsraft < Formula
+  desc "Terminal feed reader"
+  homepage "https://codeberg.org/newsraft/newsraft"
+  url "https://codeberg.org/newsraft/newsraft/archive/newsraft-0.29.tar.gz"
+  sha256 "1dbec925d5556cae70e92dcc7b69cc4899473deb1d26a600b8903dbc0b3fde65"
+  license "ISC"
+
+  depends_on "scdoc" => :build
+  depends_on "gumbo-parser"
+  # newer ncurses required on older macOS versions
+  depends_on "ncurses"
```

Comment:
Do we know which versions? If so, we can use `uses_from_macos "ncurses", since: :foo`. This would allow us to only install brewed `ncurses` on systems that need it. We also do not need the comment.

Reply:
@p-linnane at first I thought Sequoia, but after failed build I looked closely, and it's still lower than required 6.1, so I removed the comment and left it as is.